data/human/human missions.txt

```diff
@@ -6849,3 +6849,320 @@ mission "FW Refugees to Humanika"
 
+mission "Adelita 1"
+	name "Exhibition installations to <planet>"
+	description `Transport <tons> of exhibition installations to <destination> for the Adelita collective. Payment is <payment>.`
+	minor
+	source
+		attributes "paradise"
+		not attributes "mining"
+		not attributes "textiles"
+		not attributes "research"
+		not attributes "factory"
+	destination "Moonshake"
+	cargo "exhibition installations" 5
+	to offer
+		random < 80
+		has "Expedition to Hope 2: done"
+		"combat rating" > 40
+
+	on offer
+		conversation
+			`Walking around <origin>'s luxurious spaceport, you notice a significant group of people gathered around one of the larger hangars; there are prominent signs that say "The Evacuation of Hope" in bold letters by it. It looks like some sort of exhibition.`
+			choice
+				`	(Check it out.)`
+				`	(Not right now.)`
+					defer
+
+			`	You approach some of the people at the end of the line and ask what's going on. "It's Adelita's exhibition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their exhibitions are always free, so this is a great chance to see it if you haven't already."`
+			choice
+				`	"Sure, why not."`
+				`	"Maybe some other time."`
+					defer
+
+			`	"You're going to love it," she says. "No one knows who they really are, but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it."`
+			`	You reach the entrance to the exhibition soon enough. Inside is a large room filled with photos, videos, interviews and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise Worlds that protested the evacuation. At one end of the hangar, there is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and the Republic Navy; some of them quite crude and tasteless. It's a strong rebuke to see those displayed here, on a Paradise Planet.`
+			`	There are many, many photos, but one catches your eye. It's of a woman standing in line to board a ship, a baby held at her hip and a look of sad determination on her face. Their cheeks, rosy and chapped, stand out in contrast with the cold blues in the background.`
+			`	You head towards the exit, but as the automatic doors open, an arctic blast rushes at you from every direction. Instead of the spaceport's open apron, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries gradually being covered by ash in a snowy field near a small town as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
```

Comment:
```suggestion
			`	You head towards the exit, but as the automatic doors open, an arctic blast rushes at you from every direction. Instead of the spaceport's open apron, you find yourself in a life-sized 3D representation of the evacuation of Hope, complete with simulated climate effects and a projected horizon. There are huge Navy Cruisers and Auxiliaries in a snowy field near a small town gradually being covered by ash as hundreds of refugees slowly board. But what stands out most is the massive volcano erupting in the background.`
```
Also maybe break that last sentence `But what stands out most is the massive volcano erupting in the background.` out into a new line?

Reply:
Hmm, would it be too dramatic for it to be out on another line?

It is a massive volcano...